### PR TITLE
Rename log and restore newlines

### PR DESCRIPTION
**Pull request by vmihailenco**
@Codebreaker101 PTAL

### CODE REVIEW

File: migrate/migrations.go [hunk 1]
```diff
@@ -439,16 +440,16 @@ func (m *Migrations) genMigrationName(name string) (string, error) {
 }
 
 func (m *Migrations) runUp(ctx context.Context, db *bun.DB, migration *Migration) error {
-	m.log.Printf("running migration %s... ", migration.Name)
+	m.printer.Printf("running migration %s... ", migration.Name)
 	if migration.Up != nil {
 		if err := migration.Up(ctx, db); err != nil {
-			m.log.Printf("unable to run migration %s: %s", migration.Name, err)
+			m.printer.Printf("failure: %s\n", err)
```
Comment (vmihailenco) [inline migrate/migrations.go:446]: No need to repeat the migration twice, because the full log string is `running migration %s... failure: %s\n`

**Comment by Codebreaker101** [comment]
TBH I'm not happy with the state of the logger, or in this case _status printer_. My initial PR where a user can se a logging function was primarily to be able to turn off any printing whilst keeping current functionality. Also, I was deterred from making large changes since #49 would probably break that. 

From what I can tell, this package does not have any logging, only status printing. It was my mistake for thinking that the `Printf`s were logging, hence the naming of vars and functions in my PR. If we keep in mind that this is a package, what can be done? 

Let's not make any assumptions about the user -- if a user wants any printing, they should enable it. No package should print anything unless told otherwise. It is ok for bun-starter-kit, since it is an app, but not from a migration package, since in most cases (my opinion) this package will be integrated into other apps and any printing is done by the app. If looking from the context of logging, new lines, tabs and such should be avoided since they don't work well with logging packages. 

We need a way for user to have access to migration/rollback status during the execution of the actions. To do that we could implement some kind of hooks:

``` go
// general idea 
type MigrationHooks interface {
	BeforeMigrateRun(group int64, migrations []Migration)
	AfterMigrateRun(group int64, migrations []Migration)

	BeforeMigrate(migration *Migration)
	AfterMigrate(migration *Migration, err error)
	
	BeforeRollbackRun(group int64, migrations []Migration)
	AfterRollbackRun(group int64, migrations []Migration)
	
	BeforeRollback(migration *Migration)
	AfterRollback(migration *Migration, err error)
}
```
That was, status printing can be implemented in the bun-starter-kit, any user is free to implement their own and migration package is clean from any printing.

**Comment by vmihailenco** [comment]
Good default logging is an essential feature for migrate package. I understand that some users want to customize it, but the majority will not bother.

I believe that logging should be enabled by default because it helps new users understand what is happening. Once they get used to it, they may want to disable all logging. But first they must get used to it.

Perhaps we can keep all the logging from migrate package and move it to the starter-kit. That makes the starter-kit even more important/essential, but perhaps that is a good thing...

**Comment by vmihailenco** [comment]
@Codebreaker101 please take another look

**Comment by Codebreaker101** [comment]
@vmihailenco looking good! Migrator returns its status after run so no need for hooks. Great!

**Comment by vmihailenco** [comment]
Thanks for the feedback. Merging then :)

**Pull request merged**